packaging/build_wheels.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash -eux
+
+#	L I B R A R Y
+
+build_wheel() { (
+	# runs isolated in a sub-shell.
+
+	local runtimedir="$1"
+	local sdist="$2"
+
+	# Activate runtime in a subshell
+	PATH="$runtimedir/bin:$PATH"
+	hash -r
+
+	local runtime="${runtimedir##*/}"
+
+	# Build wheel from source tarball
+	pip --verbose wheel --no-deps --wheel-dir "$distdir" "$sdist"
+	wheel="$distdir/$tarname-$runtime-linux_$AUDITWHEEL_ARCH.whl"
```

Comment:
You can query the current runtime with `setuptools.pep425tags` if you want:
```
>>> from setuptools import pep425tags
>>> pep425tags.get_impl_tag()
'cp37'
>>> pep425tags.get_abi_tag()
'cp37m'
>>> pep425tags.get_platform()
'linux_x86_64'
```

Reply:
Sadly, setuptools dropped pep425tags module. It's down deep in its _vendor directory. However, I found that `distutils.util` has a `get_platform`. I updated to remove the hardcoded `linux`. Thanks for the point !

Reply:
Yeah, you're right; it's actually quite hilarious how every tool tries hiding this info (first `pip.pep425tags` moved to `pip._internal`, now `setuptools` moves it to vendored stuff). There is also `packaging.tags` available, but it has to be installed separately; the one vendored with `setuptools` is stripped AFAIK.